### PR TITLE
refactoring and networkx fix

### PR DESCRIPTION
**Pull request by setten**
refactoring gwwrapper

pep8 corrections
some tabs to spaces 
fix for the colors in the network plots

### CODE REVIEW

File: monty [hunk 1]
```diff
@@ -0,0 +1 @@
+Subproject commit 6ea7e10aec9b20910a84e37d0eaa551d1ff756da
```
Comment (gmatteo) [inline monty]: ?

Comment (setten) [inline monty]: no clue what this is 
should I remove it?

File: pymatgen/io/vasp/inputs.py [hunk 2]
```diff
@@ -1535,6 +1535,7 @@ def from_symbol_and_functional(symbol, functional="PBE"):
         for p in paths_to_try:
             p = os.path.expanduser(p)
             p = zpath(p)
+            print(p)
```
Comment (gmatteo) [inline pymatgen/io/vasp/inputs.py]: ?

File: pymatgen/io/vaspio/vasp_output.py [hunk 3]
```diff
@@ -18,7 +18,2175 @@
 
 
 import warnings
+<<<<<<< HEAD
```
Comment (gmatteo) [inline pymatgen/io/vaspio/vasp_output.py]: ?

File: monty (hunk 1 above)
Comment (gmatteo) [inline monty]: yes

File: pymatgen/io/vasp/inputs.py (hunk 2 above)
Comment (setten) [inline pymatgen/io/vasp/inputs.py]: removed

File: pymatgen/io/vaspio/vasp_output.py (hunk 3 above)
Comment (setten) [inline pymatgen/io/vaspio/vasp_output.py]: removed

**Pull request merged**